dcc-portal-ui/app/scripts/app/js/app.js

```diff
@@ -611,7 +611,7 @@
 
     // If invalid route is requested
     $urlRouterProvider.otherwise(function ($injector, $location){
-      const redirect = _.find(REDIRECTS, (redirect) => redirect.from === $location.path());
+      const redirect = _.find(REDIRECTS, (redirect) => _.startsWith($location.path(), redirect.from));
```

Comment:
Can you give me an example? I need something that I can inject inside `app.config`